src/System.Management.Automation/engine/runtime/Operations/MiscOps.cs

```diff
@@ -2145,6 +2145,11 @@ internal static Type ResolveTypeName(ITypeName typeName, IScriptExtent errorPos)
 
         internal static bool IsInstance(object left, object right)
         {
+            if (right is null)
+            {
+                return left is null;
```

Comment:
Is there a reason this code path differs from the `IsNotOperator` method in not returning boxed true/false values? (That's what those are in the other method, I think?)

Reply:
I don't have that history, so I cannot say for sure. I just followed what was already being done in each of the methods that was updated, to make sure that I returned the same type of objects that were already being returned.

Reply:
Gotcha. Yeah, that makes sense, looking more closely at the surrounding code...

Probably consumed internally and not user-facing or something. 🙂